Logging.sln

```diff
@@ -1,9 +1,9 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 14
-VisualStudioVersion = 14.0.23107.0
+# Visual Studio 2013
+VisualStudioVersion = 12.0.40629.0
```

Comment:
I was adding sharep projects that is much easier to do in VS 2013 where I have an extension for that. And it will open in both VS.

---
In reply to: [66371203](https://github.com/Microsoft/ApplicationInsights-dotnet-logging/pull/34#discussion_r66371203) [](ancestors = 66371203)